package.json

```diff
@@ -9,7 +9,7 @@
     "build:laboratory": "cd app/resonant-laboratory && webpack",
     "build:laboratory:tests": "webpack --config ./app/resonant-laboratory/webpack-testing.config.js",
     "test:laboratory:unit": "npm run test:laboratory:unit:raw | tap-spec",
-    "test:laboratory:unit:raw": "node build/resonant-laboratory/tests.js",
+    "test:laboratory:unit:raw": "for t in `ls build/resonant-laboratory/*.tests.js`; do node $t; done",
```

Comment:
Oh I see. I think you can pass multiple files directly to `node` but maybe that's not right.

This isn't that big of a deal, it's just that it depends on a unix-type shell to work properly.